selenium_tests/management/commands/snapshot_dashboard_states.py

```diff
@@ -154,6 +159,16 @@ def with_coupon(self, amount_type, is_program, is_free):
         )
         UserCoupon.objects.create(user=self.user, coupon=coupon)
 
+    def with_coupon_no_financial_aid_application(self):
+        """Coupon without a financial aid application"""
+        self.with_coupon(Coupon.PERCENT_DISCOUNT, True, False)
+        FinancialAid.objects.all().delete()
```

Comment:
At this point there should only be one user with financial aid but I'll add a filter on that user to make sure